### DISCUSSION TITLE
创建juicesfs PVC之后显示错误,一直处于Pending状态

### DISCUSSION THREAD

**Discussion by garywu2018** [juicedata/juicefs#1979] [Q&A] [answered]
<img width="1452" alt="image" src="https://user-images.githubusercontent.com/39235547/167448191-2378390e-d65b-42c7-9542-69cedd584526.png">

**Comment by xiaogaozi** [comment, marked as answer]
你需要查看 JuiceFS CSI Driver 的日志以了解具体原因，可以参考这个文档：https://juicefs.com/docs/zh/csi/troubleshooting
